src/testRunner/compilerRunner.ts

```diff
@@ -127,44 +127,32 @@ export class CompilerBaselineRunner extends RunnerBase {
 
 class CompilerTest {
     private static varyBy: readonly string[] = [
-        "allowArbitraryExtensions",
-        "allowImportingTsExtensions",
-        "allowSyntheticDefaultImports",
-        "alwaysStrict",
-        "downlevelIteration",
-        "experimentalDecorators",
-        "emitDecoratorMetadata",
-        "esModuleInterop",
-        "exactOptionalPropertyTypes",
-        "importHelpers",
-        "importHelpers",
-        "isolatedModules",
-        "jsx",
-        "module",
-        "moduleDetection",
-        "moduleResolution",
+        // implicit variations from defined options
+        ...ts.optionDeclarations

```

Comment:
Thank you for doing this, I was getting really tired of adding new flags and then wondering why I couldn't vary tests on them